R/sem.R

```diff
@@ -1054,12 +1107,23 @@ checkLavaanModel <- function(model, availableVars) {
     gettext("Relative Noncentrality Index (RNI)")
   )
   if (length(options[["models"]]) == 1) {
-    fitin[["value"]] <- fmli[[1]][c("cfi", "tli", "nnfi", "nfi", "pnfi", "rfi", "ifi", "rni")]
+    fitin[["value"]] <- fmli[[1]][c("cfi", "cfi.t", "tli", "nnfi", "nfi", "pnfi", "rfi", "ifi", "rni")]
+    cfi_t_footnote <- paste0(gettextf("The T-size equivalents of the conventional CFI cut-off values (poor < 0.90 < fair < 0.95 < close) are <b>poor < %s < fair < %s < close</b> for model: ",
```

Comment:
this footnote should be all formatted with a `gettextf` + use positional arguments (see https://github.com/jasp-stats/INTERNAL-jasp/issues/2125)